### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by aelkady**

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -1,6 +1,88 @@
+<center>
+
 # RichTextView
+
+![alt text](../website/static/img/rtv.png "Title")
+
 [![Builds](https://img.shields.io/circleci/project/github/tophat/RichTextView.svg)](https://circleci.com/gh/tophat/RichTextView)
```
Comment (jakebolam) [inline .github/README.md]: Add cocapods monthly downloads badge:
`
[CocoaPods](https://cocoapods.org/pods/THRichTextView)`

File: .github/README.md [hunk 2]
```diff
@@ -1,6 +1,88 @@
+<center>
+
 # RichTextView
+
+![alt text](../website/static/img/rtv.png "Title")
```
Comment (jakebolam) [inline .github/README.md]: First badge should be cocoapods version badge:
`
[Cocoapods](https://cocoapods.org/pods/THRichTextView)`

File: .github/README.md (hunk 1 above)
Comment (aelkady) [reply .github/README.md]: Done

File: .github/README.md (hunk 2 above)
Comment (aelkady) [reply .github/README.md]: Done

**Pull request merged**